### PR TITLE
Completed TL translations for client-tfs.asc - 1,428 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
I have completed the translations for the file client-tfs.asc. Please help me review my work. Thank you.

### CODE REVIEW

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 1]
```diff
@@ -203,16 +203,17 @@ $ git log --oneline --graph --decorate --all
...
-  Be aware, however, that TFVC doesn't think this way, and if you push merge commits your history will start to look different on both sides, which can be confusing.
-  However, if you plan on submitting all of your changes as one changeset, this is probably the easiest choice.
-. Rebasing makes our commit history linear, which means we have the option of converting each of our Git commits into a TFVC changeset.
-  Since this leaves the most options open, we recommend you do it this way; git-tf even makes it easy for you with `git tf pull --rebase`.
+. Ang paggawa ng isang merge na commit ay nagdaramdam na natural bilang isang user ng Git (kahit na, iyon ay ang kung ano ang ginagawa ng `git pull`), at ang git-tf ay maaaring gawin ito para sa iyo gamit ang isang simpleng `git tf pull`.
+  Magbigay alam, gayunpaman, na ang TFVC ay hindi nag-iisip sa ganitong paraan, at kung ikaw ay magpu-push ng merge na mga commit ang iyong kasaysayan ay magsisimulang magmukhang kakaiba sa parehong banda, na maaaring nakakalito.
+  Gayunpaman, kung plinano mong magsumite ng lahat ng iyong mga pagbabago bilang isang hanay ng pagbabago, ito na yata ang pinakamadaling mapagpipilian.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: plinano to pinaplano

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 2]
```diff
@@ -255,22 +256,22 @@ $ git log --oneline --graph --decorate --all
...
-There are a couple of other considerations you'll want to keep in mind:
+Ito ay ang pangunahing daloy ng trabaho.
+Mayroong iilang ibang konsiderasyon na gugustuhin mong isaisip:
 
-* There is no branching.
-  Git-tf can only create Git repositories from one TFVC branch at a time.
-* Collaborate using either TFVC or Git, but not both.
-  Different git-tf clones of the same TFVC repository may have different commit SHA-1 hashes, which will cause no end of headaches.
-* If your team's workflow includes collaborating in Git and syncing periodically with TFVC, only connect to TFVC with one of the Git repositories.
+* Walang pag-branch.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc]: Walang pag-branch to Wala nang pag-branch

File: book/09-git-and-other-scms/sections/client-tfs.asc [hunk 3]
```diff
@@ -328,8 +329,8 @@ PS> git log --all --oneline --graph --decorate
 * b75da1a New project
 ----
 
-Now we're ready to complete our contribution by checking in our code to the TFVC server.
-We'll use the `rcheckin` command here to create a TFVC changeset for each Git commit in the path from HEAD to the first `tfs` remote branch found (the `checkin` command would only create one changeset, sort of like squashing Git commits).
+Ngayon tayo ay handa nang kumpletuhin ang ating pag-ambag sa pamamagitan ng pagtsi-check in ng ating code sa TFVC na server.
+Gagamitin natin ang `rcheckin` na utos dito upang lumikha ng isang TFVC na hanay ng pagbabago para sa bawat Git na commit sa landas mula sa HEAD patungo sa unang `tfs` na remote na branch na natagpuan (ang `checkin` na utos ay lilikha lamang ng isang hanay ng pagbabago, tulad ng pagpipi ng Git na mga commit).
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-tfs.asc:332]: lilikha  to lumikha

Comment (josteinfrancois) [reply book/09-git-and-other-scms/sections/client-tfs.asc:332]: will retain the phrase `lilikha lamang ng isang`, to emphasize future tense equivalent for `would only create`

**Comment by josteinfrancois** [comment]
Hello @robin-ho . Glad you reviewed my work. Updated already.

**Pull request merged**